testcases/PowerNVDump.py

```diff
@@ -724,11 +732,13 @@ def runTest(self):
 
 class KernelCrash_KdumpSMT(PowerNVDump):
 
+    #This test tests kdump/fadump with smt=1,2,4
+
     def runTest(self):
         self.cv_SYSTEM.goto_state(OpSystemState.OS)
         for i in ["off", "2", "4"]:
             self.setup_test()
-            self.c.run_command("ppc64_cpu --smt=%s" % i)
+            self.c.run_command("ppc64_cpu --smt=%s" % i, timeout=600)
```

Comment:
this also go in another commit or having valid commit message and why it require